shared/repos/GsDevKit_launcher/rowan/filetree/gsdevkit_launcher-Scripts.package/GdkL_Products.class/instance/main.st

```diff
@@ -0,0 +1,8 @@
+other
+main
+	| resultStream |
+
+	resultStream := String new writeStream.
+	self installedProductsReportOn: resultStream.
+
+	^resultStream contents
```

Comment:
For the gsdevkit_launcher scripts, we should be using `self stderr` for text output. This is something that I haven't emphasized, but `stdout` is reserved for writing objects in STON format (try putting `self objOut nextPut: Array new` in #main) so that scripts will be able to read/write objects on `objIn` and `objOut`. The object return by #main is also written to `objOut`... so text and user messages should be written to stderr ...